### PR TITLE
Update Dockerfile for consistency to resolve the wrong performance comparison result

### PR DESCRIPTION
**Pull request by oubotong**
Signed-off-by: Botong Ou <richard97@g.ucla.edu>

Make Dockerfile consistent with the configuration of Envd. The changes include:
1. Use v0.3.11 envd-sshd which is used in Envd
2. Add `apt-get install -y apt-utils` as Envd also installed this [system.go](https://github.com/tensorchord/envd/blob/main/pkg/lang/ir/v1/system.go#:~:text=sb.WriteString(%22apt%2Dget%20update%20%26%26%20apt%2Dget%20install%20%2Dy%20apt%2Dutils%20%26%26%20%22))
3. Use the same version of conda as used in Envd and make the installation steps the same

Using this updated Dockerfile should give more accurate results for the performance comparison (Around *83s* for docker-cpu and *95s* for envd-v1 on my local machine).

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -30,43 +30,43 @@ RUN apt-get update && \
...
+    sh miniconda.sh -b -u -p /opt/conda && \
+    touch ~/.bashrc && \
     echo ". /opt/conda/etc/profile.d/conda.sh" >> ~/.bashrc && \
     echo "conda activate base" >> ~/.bashrc && \
+    echo -e "channels:\n  - defaults" > /opt/conda/.condarc && \
     find /opt/conda/ -follow -type f -name '*.a' -delete && \
     find /opt/conda/ -follow -type f -name '*.js.map' -delete && \
-    /opt/conda/bin/conda clean -afy
+    /opt/conda/bin/conda clean -afy && \
+    rm miniconda.sh
```
Comment (kemingy) [inline Dockerfile]: already removed in line 63

Comment (oubotong) [reply Dockerfile]: Fixed

File: Dockerfile [hunk 2]
```diff
@@ -30,40 +30,40 @@ RUN apt-get update && \
...
     wget "${MINICONDA_URL}" -O miniconda.sh -q && \
     echo "${SHA256SUM} miniconda.sh" > shasum && \
     if [ "${CONDA_VERSION}" != "latest" ]; then sha256sum --check --status shasum; fi && \
-    mkdir -p /opt && \
-    bash miniconda.sh -b -p /opt/conda && \
+    mkdir -p /opt/conda && \
+    sh miniconda.sh -b -u -p /opt/conda && \
     rm miniconda.sh shasum && \
-    ln -s /opt/conda/etc/profile.d/conda.sh /etc/profile.d/conda.sh && \
+    touch ~/.bashrc && \
```
Comment (kemingy) [inline Dockerfile:63]: What's the purpose of this? I think it's run as the root user.

Comment (oubotong) [reply Dockerfile:63]: True, I am not sure why touch `~/.bashrc` here but it's used in the  [install_conda.sh](https://github.com/tensorchord/envd/blob/main/pkg/lang/ir/v1/install_conda.sh#:~:text=touch%20~/.bashrc%20%26%26%20%5C) in envd v1

Comment (oubotong) [reply Dockerfile:63]: > What's the purpose of this? I think it's run as the root user.

True, I am not sure why touch ~/.bashrc here but it's used in the [install_conda.sh](https://github.com/tensorchord/envd/blob/main/pkg/lang/ir/v1/install_conda.sh#:~:text=touch%20~/.bashrc%20%26%26%20%5C) in envd v1.

**Review by kemingy — APPROVED** [review]

**Pull request merged**